scene/gui/color_picker.cpp

```diff
@@ -689,7 +705,7 @@ void ColorPicker::_text_type_toggled() {
 		text_type->set_icon(nullptr);
 
 		c_text->set_editable(true);
-		c_text->set_tooltip_text(RTR("Enter a hex code (\"#ff0000\") or named color (\"red\")."));
+		c_text->set_tooltip_text(ETR("Enter a hex code (\"#ff0000\") or named color (\"red\")."));
```

Comment:
Is it intentional that some of the tooltips use `atr(ETR())` and others just `ETR()`?

Reply:
Tooltips obey auto translation, so if they are a static string, they don't need `atr()`. However, if the string has a dynamic component, it needs to be translated directly.